src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/hmctsresponsereviewed/HmctsResponseReviewedAboutToSubmitHandler.java

```diff
@@ -106,4 +108,15 @@ private void updateDocument(DwpDocument dwpDocument, DwpResponseDocument dwpResp
         }
     }
 
+    private void setDefaultOverrideFields(SscsCaseData sscsCaseData) {
+        OverrideFields defaultOverrideFields = sscsCaseData.getSchedulingAndListingFields().getDefaultOverrideFields();
+        if (isNull(defaultOverrideFields)) {
```

Comment:
Do we not want to set the po to attend field even if the override fields aren't null?

Reply:
Yes, I agree

Reply:
fixed